### PR TITLE
New connect data layout

### LINKED ISSUE #5049: Redesign connect data screen to make it fit without scrolling
Right now, the select a file button is hidden under the scroll

### PR DESCRIPTION
**Pull request by xavijam**
Basically that, no changes in the behaviour.

screen shot 2015-09-09 at 18 28 08
screen shot 2015-09-09 at 18 27 56

Fixes #5049
REVIEWER: @javierarce @josecruz 

Also, maybe important for @csobier

### CODE REVIEW

**Comment by xavijam** [comment]
@javierarce, sorry to select you again, but you implement select marker thing and I'd like to have you here :(. sorry.

**Comment by josecruz** [comment]
Looks awesome! Just update margins to 28px and :green_heart: 

screen shot 2015-09-10 at 10 08 46

**Comment by xavijam** [comment]
If I change that, it will be changed in all sections because it is a shared element, just saying.

**Comment by xavijam** [comment]
@javierarce could you take a look "porfa-please"?

File: lib/assets/javascripts/cartodb/common/dialogs/map/image_picker/file_upload_template.jst.ejs [hunk 1]
```diff
@@ -1,32 +1,28 @@
 <form class="Form js-form">
-  <div class="ImportPanel-header"><h3 class="ImportPanel-headerTitle">
-      Upload a file or a URL
-    </h3>
-    <p class="ImportPanel-headerDescription">Paste a URL or upload a file from your computer</p>
+  <div class="ImportPanel-header">
+    <h3 class="ImportPanel-headerTitle">Upload a file or an URL</h3>
+    <p class="ImportPanel-headerDescription">
+      Paste an URL <% if (fileEnabled) { %>or select a file like JPG, GIF, PNG, SVG<% } %>
```
Comment (javierarce) [inline lib/assets/javascripts/cartodb/common/dialogs/map/image_picker/file_upload_template.jst.ejs]: It should be "Paste a URL"

**Comment by javierarce** [comment]
:+1: :+1: :+1: :panda_face:

**Comment by csobier** [comment]
(Sorry, was adding milestone for the related Doc issue :-))

**Pull request merged**